src/e2e/specs/landing.spec.ts

```diff
@@ -5,6 +5,7 @@
 import { test, expect } from "../fixtures/basePage.js";
 import { defaultScreenshotOpts } from "../utils/helpers.js";
 
+test.describe.configure({ mode: "parallel" });
```

Comment:
Q: What benefit does adding parallel mode here have?

Reply:
essentially, pw runs tests in same spec file serially, if this flag is added the tests are ran parallel thus speeding up the test